Revise the positionConfig contents (#650)
DevExpress/devextreme-documentation: api-reference/50 Common/Object Structures/positionConfig/boundary.md

---
---
##### shortDescription

> A boundary element within which the overlay element should be positioned.

Reviewer: Can you clarify what you're trying to say here?
Do you mean:
An overlay element must fit within the boundary element.

Author: No, this option specifies a boundary element outside of which the overlay element cannot be placed. For example, the boundary element is some rectangle. Then, the overlay element can only be inside this rectangle.

Reviewer: ```suggestion
A boundary element in which the overlay element must be positioned.
```

---

> The boundary element limits possible positions for the overlay element. The default boundary element is <a href="https://developer.mozilla.org/en-US/docs/Web/API/Window">Window</a>. If the actual boundaries should be narrower or wider than the boundary element, set the [boundaryOffset](/Documentation/ApiReference/Common/Object_Structures/positionConfig/boundaryOffset/). Whenever the specified position exceeds the boundaries, a [collision](/Documentation/ApiReference/Common/Object_Structures/positionConfig/collision/) occurs.

Reviewer: Change "the overlay element" to "an overlay element"

By "If the actual boundaries should be narrower or wider..." do you mean "If boundaries need to be narrower or wider..."

"Whenever the specified position exceeds the boundaries..."
I don't think a "position" can exceed a boundary. Do you mean the element at that particular position? Please revise.

Author: > By "If the actual boundaries should be narrower or wider..." do you mean "If boundaries need to be narrower or wider..."

Let's return to the example with a rectangle. So, the rectangle constitutes the boundary for the overlay element. Let's say you need to actual boundaries to be shifted a bit from the rectangle. In this case, you need to specify the boundaryOffset - it is the offset of the boundaries from their initial positions. That shifting is what I mean by making the boundaries "narrower or wider".

> "Whenever the specified position exceeds the boundaries..."
I don't think a "position" can exceed a boundary. Do you mean the element at that particular position? Please revise.

"Whenever the specified position causes the overlay element to exceed the boundaries, a collision occurs."?

Reviewer: Okay, let's say:
The boundary element limits possible positions for an overlay element. The default boundary element is <a href="https://developer.mozilla.org/en-US/docs/Web/API/Window">Window</a>. If a boundary should be narrower or wider than a boundary element, set the [boundaryOffset](/Documentation/ApiReference/Common/Object_Structures/positionConfig/boundaryOffset/). When a specified position exceeds a boundary, a [collision](/Documentation/ApiReference/Common/Object_Structures/positionConfig/collision/) occurs.

> The boundary element limits possible positions for the overlay element. The default boundary element is <a href="https://developer.mozilla.org/en-US/docs/Web/API/Window">Window</a>. If the actual boundaries should be narrower or wider than the boundary element, set the [boundaryOffset](/Documentation/ApiReference/Common/Object_Structures/positionConfig/boundaryOffset/). Whenever the specified position violates the boundaries, a [collision](/Documentation/ApiReference/Common/Object_Structures/positionConfig/collision/) occurs.

Reviewer 2: I'm not  really sure, but "exceeds the boundaries" might be a better choice than "violates the boundaries", that is mostly used in psychology.
